daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DataProcessor.scala

```diff
@@ -162,15 +167,14 @@ class DataProcessor(val ssrd: SchemaSetRuntimeData)
 
   def setTunable(tunable: String, value: String): Unit = tunablesObj = tunablesObj.setTunable(tunable, value)
   def setTunables(tunables: Map[String, String]): Unit = tunablesObj = tunablesObj.setTunables(tunables)
```

Comment:
Right agove this is this:
```scala
  def setExternalVariables(extVars: File, tunable: DaffodilTunables): Unit = {
    ssrd.variables = ExternalVariablesLoader.loadVariables(extVars, ssrd, ssrd.variables, getTunables)
  }
```
That looks like another tunable related bug I think? Tunables are passed into that method, but then they are ignore and getTunables is called to get tunables. Seems odd to ignore the tunables. Not sure where it's actually called.